### DISCUSSION TITLE
Insets in iOS

### DISCUSSION THREAD

**Discussion by Diogo-Mota-Almeida** [jpudysz/react-native-unistyles#234] [Q&A] [answered]
Hello fellow devs!

Before anything, I really want to pay a compliment to all the contributors for this project. Congratulations and Thank you!

I was looking into obtaining the value of insets as it is displayed in the [Edge to Edge page](https://reactnativeunistyles.vercel.app/reference/edge-to-edge/):
![image](https://github.com/jpudysz/react-native-unistyles/assets/108743844/3644714f-47e2-4af4-ae83-178bfc4f7944)

I tested it on Android and all went well, and then I asked a colleague to try it on iOS and it also worked, which I didn't expect since on the page's documentation it is only tagged with Android:
![image](https://github.com/jpudysz/react-native-unistyles/assets/108743844/883d503c-ae68-4cca-a321-04cf3dafda45)

So here I am, opening an issue because all is working well, the irony...
My question is, is that syntax only available for Android, and I had a stroke of luck, or (more probable) am I confusing things?

**Comment by jpudysz** [comment, marked as answer]
Hey 👋 

Sorry for the confusion!

Insets are implemented for both Android and iOS, but edge-to-edge layout refers only to Android.

All iOS apps are edge-to-edge and you can't change this. On the other hand, no app is edge-to-edge by default on Android.

The only thing that doesn't work for iOS is manipulating the navigation bar because there isn't one 😅

All other APIs are cross platform! 

I hope that answers your question

**Reply by Diogo-Mota-Almeida** [reply]
Thank you for your quick reply!
Have a good week
